libs/blockscout-client/src/v2/import.rs

```diff
@@ -0,0 +1,23 @@
+use super::types;
+use crate::{Client, Result};
+
+pub mod smart_contracts {
+    pub use super::*;
+
+    pub async fn get(
+        client: &Client,
+        address_hash: ethers_core::types::Address,
+    ) -> Result<types::ImportSmartContractResponse> {
+        let path = format!(
+            "/api/v2/import/smart-contracts/0x{}",
+            hex::encode(address_hash)
+        );
+        let headers = match client.api_sensitive_endpoints_key() {
+            None => vec![],
+            Some(key) => vec![("x-api-key", key)],
+        };
```

Comment:
I would suggest to make logic of adding `x-api-key` to be more abstract to easily add this logic for new endpoints, however it's okay to leave it for now

Reply:
Could you please explain what do you mean by more abstract?

Reply:
I mean more declarative
Maybe building request like `client.get().with_sensitive_endpoints_key().with_headers(headers).build(url)`

Reply:
and `with_sensitive_endpoints_key` will handle your `match` for actual key

Reply:
but we can leave it for next version I guess ;)

Reply:
I see
I agree, that looks nice to have, but I'm also for moving it to the next iteration 👍